t/t2231-job-info-lookup.t

```diff
@@ -33,6 +33,37 @@ test_expect_success 'job-info: generate jobspec for simple test job' '
         flux jobspec --format json srun -N1 sleep 300 > sleeplong.json
 '
 
+#
+# job info lookup list keys tests
+#
+
+test_expect_success 'flux job info listing of keys works (job complete)' '
+        jobid=$(submit_job) &&
+	flux job info $jobid > list_keys1.out &&
+        grep "^J" list_keys1.out &&
+        grep "^R" list_keys1.out &&
+        grep "^eventlog" list_keys1.out &&
+        grep "^jobspec" list_keys1.out &&
+        grep "^guest.exec.eventlog" list_keys1.out &&
+        grep "^guest.input" list_keys1.out &&
+        grep "^guest.output" list_keys1.out
+'
+
+test_expect_success 'flux job info listing of keys works (job running)' '
+        jobid=`flux job submit sleeplong.json` &&
+        fj_wait_event $jobid start >/dev/null &&
+	flux job info $jobid > list_kesy2.out &&
```

Comment:
Nitpick:
```suggestion
	flux job info $jobid > list_keys2.out &&
```

Reply:
oops, thanks for the catch.  Re-pushed with a fix.  Since its just spelling errors, just squashed the fix.